content/en/synthetics/api_tests/icmp_tests.md

```diff
@@ -117,13 +117,20 @@ Click **Save** to save your test and have Datadog start executing it.
 
 You can create local variables by defining their values from one of the below available built-ins:
 
-| Pattern                    | Description                                                                                                 |
-|----------------------------|-------------------------------------------------------------------------------------------------------------|
-| `{{ numeric(n) }}`         | Generates a numeric string with `n` digits.                                                                 |
-| `{{ alphabetic(n) }}`      | Generates an alphabetic string with `n` letters.                                                            |
-| `{{ alphanumeric(n) }}`    | Generates an alphanumeric string with `n` characters.                                                       |
-| `{{ date(n, format) }}`    | Generates a date in one of our accepted formats with a value of the date the test is initiated + `n` days.        |
-| `{{ timestamp(n, unit) }}` | Generates a timestamp in one of our accepted units with a value of the timestamp the test is initiated at +/- `n` chosen unit. |
+`{{ numeric(n) }}`
```

Comment:
All the images in this files preview are broken, but working in the live docs: https://docs.datadoghq.com/synthetics/api_tests/icmp_tests/